packages/python/readme_metrics/tests/MetricsMiddleware_test.py

```diff
@@ -1,43 +1,8 @@
-import pytest  # pylint: disable=import-error
-import requests
-import json
-
-from .fixtures import Environ
+import pytest
 
 from readme_metrics import MetricsApiConfig
 from readme_metrics import MetricsMiddleware
-from readme_metrics.Metrics import Metrics
-
-# for this, I'm not exactly sure how to test the __call__ function
-# possible options I considered was making a mock server inside this test case
-# connected to the middleware somehow
-
-
-class MockServer:
```

Comment:
Removed this because it wasn't being used and we have better server integration tests elsewhere.